### DISCUSSION TITLE
Integrating with ODK

### DISCUSSION THREAD

**Discussion by pramodkvarma** [Sunbird-Saral/Community#4] [General]
We should find a community member who is interested in offering Saral as part of ODK to entire ODK community. Given [ODK](https://getodk.org/) is highly adopted across the world for data collection, Saral can add huge value to ODK adopters.

@dileep-gadiraju @surendrasinghs @rahul101001000 comments?

For folks who are new, see what Saral can offer - https://saral.sunbird.org/

**Comment by rahul101001000** [comment]
Brilliant idea @pramodkvarma. This would make ODK even more useful.

Next steps for potential contributors - 
1. Create a ODK+Saral demo. 
2. Understand ODK architecture and figure out where would Saral fit in e.g. as another widget type and/or in conjunction with the embeddable versions of [ODK](https://github.com/Samagra-Development/odk-android) and [Enketo](https://github.com/SakshamHaryana-SE/enketo) that Samagra has contributed.
3. Create a RFC and initiate a discussion at the [ODK forum](https://forum.getodk.org/c/features/9)

If anyone is interested, I and/or @ChakshuGautam could guide you. 

Regards
Rahul

**Reply by pramodkvarma** [reply]
Great! Thanks @rahul101001000

**Reply by ChakshuGautam** [reply]
Would love to contribute to this.

**Reply by pramodkvarma** [reply]
Super! Thanks @ChakshuGautam 
Maybe you can start with design thoughts here? Are you familiar with ODK?
